tests/test_work.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+import os
+import pytest
+import queue
+
+from receptor_satellite import worker
+from receptor_satellite.satellite_api import HEALTH_OK, HEALTH_CHECK_OK
+from constants import UUID
+
+
+logger = logging.getLogger(__name__)
+
+
+class FakeInnerEnvelope:
+    raw_payload = None
+
+
+@pytest.mark.skipif(
+    not os.environ.get("SATELLITE_URL", None),
+    reason="No configured Satellite"
+)
+def test_health_check_task():
+    q = queue.Queue()
+    message = FakeInnerEnvelope()
+    message.raw_payload = json.dumps(dict(foreman_uuid=UUID))
```

Comment:
Could we get the UUID from environment with fallback to constant?

Reply:
The idea is that we should sanity check that the Satellite the plugin is talking to matches the UUID of the Satellite configured in Sources. So if we fall back to an environment variable, we're eliminating that check. I'm fine with that if that's what you'd prefer - I can just imagine with node IDs and Satellite IDs flying around, it would be easy to mix them up in configuration on c.rh.c

Reply:
Now I'm slightly confused. When and where is this test supposed to be run?

Reply:
Oh, no ,sorry, that's my mistake - I misread where in the code this was :) Backing up...

Yes, you're totally right, this should absolutely come from the environment. Thank you.